Cargo.toml

```diff
@@ -29,7 +29,7 @@ chacha = [ "chacha20poly1305", "cipher" ]
 ed25519 = [ "ed25519-zebra" ]
 x25519 = [ "x25519-dalek" ]
 random = [ "getrandom" ]
-aes = [ "aes-gcm", "cipher" ]
+aes = [ "aes-crate", "aes-gcm", "cipher" ]
```

Comment:
Should there be a separate feature "aes-gcm" to enable only gcm mode? This would allow to enable different modes separately ("aes-cbc", "aes-gcm", etc.)